app/src/main/java/org/jellyfin/androidtv/ui/shared/OutlineSpan.kt

```diff
@@ -0,0 +1,65 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.text.style.ReplacementSpan;
+import androidx.annotation.ColorInt
+import androidx.annotation.Dimension
+
+/**
+ * Source: https://github.com/santaevpavel/OutlineSpan
+ *
+ * A class that draws the outlines of a text when given a stroke color and stroke width.
+ */
+
+class OutlineSpan(
```

Comment:
This class seems simple enough to write manually instead of bringing in a file from another (differently licensed) project.

Reply:
The guy maintains a library, I just took the class and I actually modified it a bit. I thought I've to mention from where I took most of it and unfortunately I don't know what you want me to do about it.

Reply:
Technically the license that code uses (Apache 2) conflicts with the license of this project (GPL 2). Additionally, the terms of the Apache 2 license require more than the simple "source: url" credit you use here.

You could ask the author(s) of the class/file to license it differently for use here, with one compatible with GPL 2.

But it may be simpler to just implement your own code or find something different that is licensed appropriately.